ansible_collections/arista/avd/docs/plugins/Filter plugins/range_expand.md

```diff
@@ -0,0 +1,76 @@
+# range_expand
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.range_expand` when using this plugin.
+
+Expand a range of interfaces or list of ranges and return as a list of strings\.
+
+## Synopsis
+
+Provides the capabilities to expand a range of interfaces or list of ranges and return as a list of strings\.
+
+The filter supports VLANs\, interfaces\, modules\, sub\-interfaces\, and ranges are expanded at all levels\.
+
+Within a single range\, prefixes \(ex\. Ethernet\, Eth\, Po\) are carried over to items without prefixes \(see examples\)\.
+
+## Parameters
+
+| Argument | Type | Required | Default | Value Restrictions | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| _input | any | True | None |  | Range as string or list of ranges\. |
+
+## Examples
+
+```yaml
+---
+- "{{ 'Ethernet1' | range_expand }}"
+# -> ["Ethernet1"]
+
+- "{{ 'Ethernet1-2' | range_expand }}"
+# -> ["Ethernet1", "Ethernet2"]
+
+- "{{ 'Eth 3-5,7-8' | range_expand }}"
+# -> ["Eth 3", "Eth 4", "Eth 5", "Eth 7", "Eth 8"]
+
+- "{{ 'et2-6,po1-2' | range_expand }}"
+# -> ["et2", "et3", "et4", "et5", "et6", "po1", "po2"]
+
+- "{{ ['Ethernet1'] | range_expand }}"
```

Comment:
Duplicate example to the first

Reply:
No this one is a list input where the first one is a string.